string image_topic
string target_type

---
conicData[] conics